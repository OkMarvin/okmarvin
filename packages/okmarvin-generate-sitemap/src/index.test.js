const generateFeed = require('./index')
test('renders correctly', () => {
  expect(
    generateFeed({
      site: {
        title: 'hello okmarvin',
        description:
          'okmarvin is a static site generator with better theme support',
        url: '/'
      },
      files: []
    })
  ).toMatchSnapshot()
})
