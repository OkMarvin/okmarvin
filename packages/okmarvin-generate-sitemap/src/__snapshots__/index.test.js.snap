// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
"<?xml version=\\"1.0\\" encoding=\\"utf-8\\"?>
<urlset xmlns=\\"http://www.sitemaps.org/schemas/sitemap/0.9\\">
</urlset>"
`;
